src/shopware_api_client/endpoints/store/core/cart.py

```diff
@@ -1,50 +1,46 @@
 from typing import Any
 
-from shopware_api_client.base import ApiModelBase, EndpointBase
+from shopware_api_client.base import ApiModelBase, EndpointBase, EndpointClass
+from shopware_api_client.endpoints.base_fields import IdField
 
 
-class LineItem(ApiModelBase["CartEndpoint"]):
+class LineItem(ApiModelBase[EndpointClass]):
     _identifier: str = "cart_line_item"
     
     payload: dict[str, Any]
     label: str
     quantity: int
-    priceDefinition: dict[str, Any]
+    price_definition: dict[str, Any]
     price: dict[str, Any]
     good: bool
     description: str
```

Comment:
Description sollte leer sein dürfen.
```suggestion
    description: str | None = None
```

Reply:
Machst du noch? Oder sehe ich das falsch?